components/wordPrompt.js

```diff
@@ -0,0 +1,57 @@
+import React, { useEffect, useState } from "react";
+import "tailwindcss/tailwind.css";
+
+const WordPrompt = () => {
+  const changeWordHandler = () => {
+    setWords((lastWords) => {
+      if (lastWords.nextWords.length != 0) {
+        return {
+          ...lastWords,
+          currentWord: lastWords.nextWords.shift(),
+        };
+      } else {
+        return {
+          ...lastWords,
+          currentWord: "Out of words!",
+        };
+      }
+    });
+  };
+  const [words, setWords] = useState({
+    currentWord: "Start",
```

Comment:
Not sure how I feel about this, this seems kinda hacky. Since we will have a start screen later on, I don't think we will need to set the first word to "start"

Reply:
Instead of having a separate `currentWord` state, can't we just try to display the last word in `nextWords`? This way you don't have to keep shifting the word into `currentWord`, and you just need to check if `nextWords` is empty when displaying the value. This also separates UI logic from your `changeWordHandler`

Reply:
You're right, I've combined both into the same state.